src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.ts

```diff
@@ -96,11 +99,47 @@ function sortBuilds(graph: BuildGraph): BuildData[] {
   result = result.reverse();
   for (const node of result) {
     node.isToolset = isToolset(node.build, graph);
+    node.hasIncoherentDependencies = hasIncoherentDependencies(node.build, result, false);
+    node.hasIncoherentDependenciesIncludingToolsets = hasIncoherentDependencies(node.build, result, true);
   }
 
   return result;
 }
 
+function hasIncoherentDependencies(build: Build, buildData: BuildData[], includeToolsets: boolean): boolean {
+  let y = buildData.find(x => x.build.id == build.id);
```

Comment:
>y [](start = 6, length = 1)

I think "y" and "z" would be more accurately named "buildData" and "depBuildData"

Reply:
Yep, derp, changed them to be a bit more meaningful :)